agents/rust/aries-vcx-agent/Cargo.toml

```diff
@@ -3,11 +3,23 @@ name = "aries-vcx-agent"
 version = "0.1.0"
 edition = "2021"
 
+[lib]
+path = "src/lib.rs"
+doctest = false
+
 [dependencies]
 failure = "0.1.8"
 serde = "1.0.145"
 aries-vcx = { path = "../../../aries_vcx" }
+messages = { path = "../../../messages" }
 derive_builder = "0.11.2"
 serde_json = "1.0.85"
 log = "0.4.17"
-uuid = "1.2.1"
+uuid = { version = "1.2.1", features = ["v4"] }
+
+[features]
+test_utils = [ "aries-vcx/test_utils" ]
+agency_pool_tests = ["test_utils" ]
```

Comment:
There is no need for adding a feature as long as there is a single test; also, I guess all tests in aries-vcx-agent will be integration tests, so maybe we can do with good old `#[cfg(test)]` annotation?